src/Prima/Pyxis/Form/Radio.elm

```diff
@@ -263,6 +275,9 @@ buildAttributes model ((Radio config) as radioModel) choice =
         options =
             computeOptions radioModel
 
+        hasValidations =
+            List.length options.validations > 0 && readerChecker model radioModel
```

Comment:
sostituire qui l'or con l'and, ha l'effetto di non mostrare mai la classe is-valid per dei campi senza validazione.

Reply:
Esattamente, infatti in teoria sarebbe corretto cosi, ma da valutare